app/admin/users.rb

```diff
@@ -0,0 +1,104 @@
+ActiveAdmin.register User do
+  after_create do |user|
+    # set role after creation
+    case params[:user][:role_type]
+    when 'Administrator'
+      user.role = Administrator.create
+    when 'Therapist'
+      user.role = Therapist.create
+    when 'Donor'
+      user.role = Donor.create
+    end
```

Comment:
Hmm... I don't see the harm in adding a few lines to handle that case, even if it shouldn't ever happen - because if by some reason a user without a role _is_ created, it would be confusing if the user is just scrapped without any indication